### PR TITLE
fix(Puray): rename puray to anim

### PR DESCRIPTION
**Pull request by devnesi**
## Description 

New domain

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
Show proof of the presence working.

File: websites/A/Anim.wtf/metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.9",
 	"author": {
-		"name": "Luscarvalho",
-		"id": "458776046184169473"
+		"name": "nesi",
+		"id": "476522405725077519"
 	},
-	"service": "Puray Animes",
+	"service": "Anim",
```
Comment (DarkVillager) [inline websites/A/Anim.wtf/metadata.json:7]: Folder name must match, the service name

**Comment by DarkVillager** [comment]
There are things broken within the presence, so fix that otherwise this PR won't get merged.

File: websites/A/Anim.wtf/metadata.json (hunk 1 above)
Comment (devnesi) [reply websites/A/Anim.wtf/metadata.json]: Renamed folder to Anim

**Comment by theusaf** [comment]
Please provide screenshots showing that the presence is working

**Comment by devnesi** [comment]
image
image
image
image
image

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**